src/components/VisualizeLinear/CorrelationDescrip/CorrelationDescrip.css

```diff
@@ -0,0 +1,10 @@
+.CorrelationDescrip{
```

Comment:
This CSS class seems to make the Graph B button disappear sometimes. I tried getting rid of it and I think it looks alright, for now.

For later: I'm thinking we should make a general CSS class for this type of text, since it's a format we will probably be using a lot. (or maybe there's a Material UI way to do it)